Description: Comment out definition of round, which conflicts with
 math.h declaration
Author: Scott Howard <showard@debian.org>
Bug-Debian: http://bugs.debian.org/633659
--- arduino-0022+dfsg.orig/hardware/arduino/cores/arduino/wiring.h
+++ arduino-0022+dfsg/hardware/arduino/cores/arduino/wiring.h
@@ -76,7 +76,7 @@ extern "C"{
 #define max(a,b) ((a)>(b)?(a):(b))
 #define abs(x) ((x)>0?(x):-(x))
 #define constrain(amt,low,high) ((amt)<(low)?(low):((amt)>(high)?(high):(amt)))
-#define round(x)     ((x)>=0?(long)((x)+0.5):(long)((x)-0.5))
+//#define round(x)     ((x)>=0?(long)((x)+0.5):(long)((x)-0.5))
 #define radians(deg) ((deg)*DEG_TO_RAD)
 #define degrees(rad) ((rad)*RAD_TO_DEG)
 #define sq(x) ((x)*(x))
